src/assets/text/droughtNarrations_desktop.js

```diff
@@ -5,54 +5,108 @@ export default {
         title: 'How to read the timeline',
         start_date: '1920-04-01', // Date marking start of annotation. Determines y trigger start position on deskop and order
         end_date: '1928-12-31', // Date marking end of annotation. Determines y trigger end position on deskop
-        text: 'Each bar represents one of the 2000 most severe streamflow drought events measured at a USGS  streamgage. The taller and darker the bar, the longer the drought. Where the chart is wider, many streamgages were experiencing drought simultaneously. The map above shows the gages with one of the 2000 most severe droughts in the record.', // Narrative text
+        text: 'Each bar represents one of the 2000 most severe streamflow drought events measured at a USGS  streamgage. The taller and darker the bar, the longer the drought. Where the chart is wider, many streamgages recorded drought conditions simultaneously. The map above shows the gages with one of the 2000 most severe droughts in the record.', // Narrative text
```

Comment:
```suggestion
        text: 'This timeline shows the 2,000 most severe streamflow drought events at a USGS streamgages over the last 100 years. Each bar shows the duration of drought at a single gage so that the taller and darker the bar, the longer the drought event. When multiple streamgages experienced drought conditions simultaneously, the chart is wider. The map above shows the gage locations for the 2,000 most severe droughts in the record.', // Narrative text
```

Reply:
I've had numerous people ask me what the x-axis is, so I think we need to be clearer on that

Reply:
I'm concerned that this might make this narration too long for small laptop screens, but I'll ask HCD to gather screenshots if she can.

Reply:
This is essential to understand the chart. It can always scroll, and you can test different browsers and screen sizes using the inspector.